ui/ui.go

```diff
@@ -144,17 +144,40 @@ func newModel(cfg Config) tea.Model {
 		cfg: cfg,
 	}
 
-	return model{
+	m := model{
 		common: &common,
 		state:  stateShowStash,
 		pager:  newPagerModel(&common),
 		stash:  newStashModel(&common),
 	}
+
+	info, _ := os.Stat(cfg.Path)
+	if info.IsDir() {
+		m.state = stateShowStash
+	} else {
+		m.state = stateShowDocument
+		m.pager.currentDocument = markdown{
+			localPath: cfg.Path,
+			Note:      stripAbsolutePath(cfg.Path, ""),
+			Modtime:   info.ModTime(),
+		}
+	}
```

Comment:
Here `info` can be nil on error.
```suggestion
	info, err := os.Stat(cfg.Path)
	if err == nil {
	      if info.IsDir() {
		      m.state = stateShowStash
	      } else {
		      m.state = stateShowDocument
		      m.pager.currentDocument = markdown{
			      localPath: cfg.Path,
			      Note:      stripAbsolutePath(cfg.Path, ""),
			      Modtime:   info.ModTime(),
		      }
	      }
	}
```